02-s3/README.md

```diff
@@ -413,17 +454,23 @@ Use your own KMS key to encrypt files in S3.
 
 _Can you use the alias when uploading files?_
 
+> Yes, but it looks like the key needs to be in the same account.
+
 ### Retrospective 2.4
 
 #### Question: Requiring Encryption
 
 _After changing your bucket policy, can you upload files that aren't encrypted?
 If so, how would you require encryption on all files?_
 
+> It looks like you still can but you can set a deny poly on put opperations that don't have encryption. 
+
```

Comment:
You should not be able to upload unencrypted files after setting a bucket policy requiring encrypted files. Any files that were previously uploaded without encryption would remain that way until re-uploaded.